content/docs/guides/saml/aad.md

```diff
@@ -20,12 +24,12 @@ The first step is to add a new Pulumi application to your Azure AD tenant.
 
     ![Enterprise applications section](/images/docs/reference/service/saml-aad/enterprise-applications.png)
 
-1. Click **New application** at the top of the **All applications** blade.
+1. Click **New application**.
 
     ![New application](/images/docs/reference/service/saml-aad/new-application.png)
 
-1. Click the **Non-gallery application** tile and in the **Add your own application** blade, enter
-   _Pulumi Cloud Console_ as the application name and click **Add**.
+1. Select **Non-gallery application** tile and in the **Add your own application** panel, enter
+   _Pulumi Cloud Console_ as the application name then click **Add**.
```

Comment:
@chrsmith This is dependent on getting #1616 merged, but can we change the instructions here to say "Pulumi Console"? When I was testing with GitHub, GitLab, and Atlassian recently, the app was still labeled "Pulumi Cloud Console".

Reply:
Yeah, we can change the text here. The "application name" in Azure AD, Okta, etc. doesn't matter. It's just for the Azure or Okta administrator to know what that SAML SSO application is for. (Read: it won't impact SAML SSO actually working.)